train.py

```diff
@@ -37,51 +39,74 @@
 X_test = zero_pad(X_test, SEQUENCE_LENGTH)
 
 # Different placeholders
-batch_ph = tf.placeholder(tf.int32, [None, SEQUENCE_LENGTH])
-target_ph = tf.placeholder(tf.float32, [None])
-seq_len_ph = tf.placeholder(tf.int32, [None])
-keep_prob_ph = tf.placeholder(tf.float32)
+with tf.name_scope('inputs'):
```

Comment:
It seems more clear to warp all input_phs into a namescope.

Reply:
OK. It seems it would be better if scopes names will have the same style (starting with capital letter if it's a word, for example)